website/docs/docs/build/udfs.md

```diff
@@ -43,7 +43,16 @@ Refer to [Function properties](/reference/function-properties) or [Function conf
 
 ## Defining UDFs in dbt
 
-You can define SQL and Python UDFs in dbt. Note: Python UDFs are currently supported in Snowflake and BigQuery when using <Constant name="core" />. Support for Python UDFs in <Constant name="fusion" /> is not yet available.
+You can define SQL and Python UDFs in dbt. Python UDFs are currently supported in Snowflake and BigQuery when using <Constant name="core" />. 
+
+:::tip
+Keep the following considerations in mind when developing UDFs:
+
+- Python UDFs aren't yet supported in <Constant name="fusion" />.
+- Additional languages (for example, Java, JavaScript, Scala) aren't currently supported.
+
+See the [Limitations](#limitations) section below for the full list of currently supported UDF capabilities.
```

Comment:
This section is more about limitations than what is supported. It might be helpful to readers to put a lot of this information in the prerequisites up top than in a limitations section at the bottom

"You must use SQL UDFs on Fusion (Python UDF support coming soon)" for example
```suggestion
See the [Limitations](#limitations) section below for the full list of currently supported UDF capabilities.
```